ssg/build_remediations.py

```diff
@@ -6,16 +6,31 @@
 import os.path
 import re
 import codecs
+from collections import defaultdict
 
+from .jinja import process_file as jinja_process_file
 from .xml import ElementTree
 from .products import parse_name, multi_list, map_name
 
+REMEDIATION_MAP = {
```

Comment:
I would go for more descriptive `REMEDIATION_TO_EXTENSION_MAP`, so it is clear what is mapped to remediations.